CMakeLists.txt

```diff
@@ -139,8 +139,10 @@ enable_cppcheck()
 # Pull in helper macros for working with leatherman libraries
 include(leatherman)
 
+file(GLOB_RECURSE ALL_SOURCES lib/src/*.cc lib/inc/*.hpp lib/inc/version.h exe/*.cc exe/*.hpp exe/*.h)
```

Comment:
you can probably do something like ${CMAKE_PROJECT_DIR}/etc/etc for each of these so that it can be moved into the locale/CMakeLists.txt file